terraform/amazon/templates/inputs.tf.template

```diff
@@ -113,6 +113,9 @@ variable "vault_root_size" {
 variable "vault_data_size" {
   default = "10"
 }
+variable "vault_volume_encrypted" {
+  default = "false"
+}
```

Comment:
Use the encrypted directly in the templates, as almost all nodes/instances should be templated in our terraform code